rmgpy/cantherm/kinetics.py

```diff
@@ -144,10 +144,12 @@ def generateKinetics(self,Tlist=None):
             tunneling.E0_TS = (self.reaction.transitionState.conformer.E0.value_si*0.001,"kJ/mol")
             tunneling.E0_prod = (sum([product.conformer.E0.value_si for product in self.reaction.products])*0.001,"kJ/mol")
         elif tunneling is not None:
-            raise ValueError('Unknown tunneling model {0!r}.'.format(tunneling))
-        
-        logging.info('Generating {0} kinetics model for {0}...'.format(kineticsClass, self.reaction))
-        
+            if tunneling.frequency is not None:
+                # Frequency was given by the user
+                pass
+            else:
+                raise ValueError('Unknown tunneling model {0!r} for reaction {1}.'.format(tunneling, self.reaction))
+        logging.debug('Generating {0} kinetics model for {0}...'.format(kineticsClass, self.reaction))
```

Comment:
Should the second `{0}` be a `{1}` ? Codacy is flagging this. Once this is fixed I can merge everything in